internal/pkg/metadb/metadb.go

```diff
@@ -522,3 +522,26 @@ func (m *MetaDB) ListBlobRefsByStatus(ctx context.Context, status blobref.BlobRe
 	iter := blobref.NewCursor(m.client.Run(ctx, query))
 	return iter, nil
 }
+
+func (m *MetaDB) GetAggregation(ctx context.Context, aggregation, storeKey, field string) (*record.Record, error) {
```

Comment:
I think we can just pass pb.AggregationType without converting it to a string. We already depend on pb in PropertyValue.